volume/volume.go

```diff
@@ -104,8 +110,9 @@ type MountPoint struct {
 	CopyData bool `json:"-"`
 	// ID is the opaque ID used to pass to the volume driver.
 	// This should be set by calls to `Mount` and unset by calls to `Unmount`
-	ID   string `json:",omitempty"`
-	Spec mounttypes.Mount
+	ID    string `json:",omitempty"`
+	Tmpfs *Tmpfs `json:"-"`
```

Comment:
@AkihiroSuda Ok, but let's not fork these into two paths. If there are options we need for docker run support, then let's add them. For now, let's get swarm-mode support for `Tmpfs` without introducing another half-baked type that has to be supported forever.